app/controllers/doorkeeper_applications_controller.rb

```diff
@@ -17,6 +17,11 @@ def update
     end
   end
 
+  def users_with_access
+    authorized_users_user_ids = @application.authorized_tokens.select(:resource_owner_id)
+    @users = policy_scope(User).where(id: authorized_users_user_ids).includes(:organisation).page(params[:page]).per(100)
```

Comment:
I think this would be _users who have ever accessed an app_. This might include users who attempted to access an app but were denied access because they don't have the "signin" permission. 

I suspect what you're really after is to see the users who currently have the "signin" permission for a given app.